<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bibliothek</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabBib">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Kunden</string>
       </attribute>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>571</width>
          <height>321</height>
         </rect>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>6</number>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>90</number>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>40</number>
        </attribute>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">4</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">5</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">6</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="mBtnLeiheBeenden">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>370</y>
          <width>111</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Zurückgeben</string>
        </property>
       </widget>
       <widget class="QPushButton" name="mDeletePersonBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>341</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Kunden entfernen und dessen Ausleihen beenden</string>
        </property>
       </widget>
       <widget class="QPushButton" name="mKundeHinzuBtn">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>150</y>
          <width>191</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Neuen Kunden hinzufügen</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="VornameEdt">
        <property name="geometry">
         <rect>
          <x>602</x>
          <y>30</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Vorname</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NachnameEdt">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Nachname</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NummerEdt">
        <property name="geometry">
         <rect>
          <x>602</x>
          <y>60</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Nummer (5-stellig)</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="mRadioBox">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>60</y>
          <width>120</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Geschlecht</string>
        </property>
        <widget class="QRadioButton" name="wRadio">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>weib&amp;lich</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="mRadio">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>mae&amp;nnlich</string>
         </property>
        </widget>
       </widget>
       <widget class="QLineEdit" name="OrtEdt">
        <property name="geometry">
         <rect>
          <x>602</x>
          <y>90</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Wohnort</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="GebdatEdt">
        <property name="geometry">
         <rect>
          <x>602</x>
          <y>120</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>DD.MM.YYYY (Geburtsdatum </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Bestand</string>
       </attribute>
       <widget class="QTreeWidget" name="BibTree">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>611</width>
          <height>192</height>
         </rect>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>125</number>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>28</number>
        </attribute>
        <attribute name="headerShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">4</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">5</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="TitelEdt">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Titel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="VerlagEdt">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Herausgeber</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="MedienNrEdt">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Nummer (vierstellig)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="AutorEdt">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Autor/ Kuenstler</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="JahrEdt">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Erscheinungsjahr</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>80</y>
          <width>120</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Medienart</string>
        </property>
        <widget class="QRadioButton" name="cRadio">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>CD</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="vRadio">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>&amp;Video</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="bRadio">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>B&amp;uch</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>20</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Neues Medium hinzufuegen</string>
        </property>
       </widget>
       <widget class="QPushButton" name="mNeuesMediumBtn">
        <property name="geometry">
         <rect>
          <x>634</x>
          <y>170</y>
          <width>251</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Hinzufügen</string>
        </property>
       </widget>
       <widget class="QPushButton" name="mAltHinzuBtn">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>270</y>
          <width>291</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Weiteres Exemplar von makiertem Medium hinzufügen</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="altNrEdt">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>240</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Nummer (vierstellig)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>270</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Makiertes Exemplar ausleihen an</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>300</y>
          <width>331</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QRadioButton" name="comboRadio">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>171</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>&amp;Kunde aus Liste auswählen</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="manuellRadio">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>20</y>
           <width>111</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>ma&amp;nuelle Eingabe</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="mLeihBtn">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>270</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Verleihen</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="manuNummerEdt">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>350</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Kundennummer</string>
        </property>
       </widget>
       <widget class="QComboBox" name="mComboBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>350</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>210</y>
          <width>311</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Neues Exemplar zu bestehendem Medium hinzufügen</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="BuchCheckbox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Bücher</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="VideoCheckbox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Videos</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="CDCheckbox">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>CDs</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="mDeleteExempBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>240</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Exemplar löschen</string>
        </property>
       </widget>
       <widget class="QPushButton" name="mDeleteProtoBtn">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>240</y>
          <width>311</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Alle Exemplare von ausgewähltem Prototyp löschen löschen</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mCloseBtn">
      <property name="text">
       <string>Speichern und beenden</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
